tests/test_slo.py

```diff
@@ -132,6 +132,17 @@ def test_SLODemultiplexer():
     assert demux.slo_watchers_by_label['slo_3'].process_datapoint.call_count == 1
     demux.slo_watchers_by_label['slo_3'].process_datapoint.reset_mock()
 
+    keyErrorRaised = False
+
+    try:
+        demux.process_datapoint(
+            props={'dimensions': {'sf_metric': '_SF_COMP_E123ABC'}}, datapoint=good, timestamp=0.0,
+        )
+    except KeyError:
+        keyErrorRaised = True
+
+    assert keyErrorRaised is False
+
```

Comment:
You could simplify this to just the `demux.process_datapoint(...)` call - the KeyError would cause the test to fail: https://stackoverflow.com/questions/20274987/how-to-use-pytest-to-check-that-error-is-not-raised

The way you have written this is more explicit, which is good, but if it does start raising a KeyError for some reason, someone's going to have to remove the try/except block anyway, to get a proper traceback so they can debug the failing test.